.kitchen.cloud.yml

```diff
@@ -0,0 +1,37 @@
+---
+driver_config:
+  openstack_username: <%= ENV['OS_USERNAME'] %>
+  openstack_api_key: <%= ENV['OS_PASSWORD'] %>
+  openstack_auth_url: <%= "#{ENV['OS_AUTH_URL']}/tokens" %>
+  key_name: <%= ENV['OS_SSH_KEYPAIR'] %>
+  private_key_path: <%= ENV['OS_PRIVATE_SSH_KEY'] %>
+  public_key_path: <%= ENV['OS_PUBLIC_SSH_KEY'] %>
+  openstack_tenant: <%= ENV['OS_TENANT_NAME'] %>
+  floating_ip_pool: <%= ENV['OS_FLOATING_IP_POOL'] %>
+  flavor_ref: <%= ENV['OS_FLAVOR_REF'] %>
+  security_groups:
+    - <%= ENV['OS_SECURITY_GROUP_NO_FIREWALL'] %>
+
+provisioner:
+  name: chef_solo
+  require_chef_omnibus: 11.12.8
```

Comment:
Which version of chef omnibus should we be using? I was sticking with it because the original .kitchen.yml was using 11.12.8.